crawl-ref/source/dat/des/variable/mini_monsters.des

```diff
@@ -2806,6 +2891,27 @@ ccA..1ccc1..Acc
        @
 ENDMAP
 
+NAME:    nicolae_burly_brawl
```

Comment:
good matrix reference, but where is the sign post????

Reply:
try to realize the truth... there is no sign post...